### PR TITLE
Feature/update member

### PR DESCRIPTION
**Pull request by FredrikMorstad**
## :sparkles: Endpoint for updating members
- Updates members based on the memberUpdate model

### MemberUpdate model
All fields are optional
- Name
- Email
- Classof
- Phone

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -27,6 +27,11 @@ class MemberInput(BaseModel):
     graduated: bool
     phone: Optional[str]
 
+class MemeberUpdate(BaseModel):
+    realName: Optional[str]
```
Comment (JoakimSjo) [inline app/models.py:31]: Shall these be snake case?

**Review by JoakimSjo — APPROVED** [review]
Left some small suggestions that could be considered. 

Anyway, LGTM

File: app/models.py [hunk 2]
```diff
@@ -27,6 +27,11 @@ class MemberInput(BaseModel):
     graduated: bool
     phone: Optional[str]
 
+class MemeberUpdate(BaseModel):
```
Comment (jonjohansen) [inline app/models.py:30]: This should probably be `MemberUpdate`?

File: app/models.py (hunk 1 above)
Comment (jonjohansen) [reply app/models.py:31]: Actually, `realName` is kind of an old token. We could probably change this fieldname in its entirety as it really doesn't make sense in this context.

**Pull request merged**